lib/v_3.0.0/templates/server/api.js

```diff
@@ -5,8 +5,12 @@
 
 const __camelName__ = require('./__camelName__Controller')
 
+// for standard projects
 const { requireLogin, requireAccountAccess } = require('../../global/handlers/authHandler')
 
+// for pro-ficiency projects
```

Comment:
rm and replace with, "ex of loading additional authentication options from external source"